opus/application/static_media/js/browse.js

```diff
@@ -999,18 +997,24 @@ var o_browse = {
                     galleryHtml += 'then click on the cart icon.  </p>';
                     galleryHtml += '</div>';
                 }
-                $(".gallery", namespace).html(galleryHtml);
+                $(".gallery", tab).html(galleryHtml);
```

Comment:
I would really like to see us break the direct connection between the URL and the DOM class. For example, the class should really be op-browse or op-cart or whatever. We should have a function that looks up the prefs.view and returns the appropriate class name.

Reply:
do you want that sort of major change to be part of this change?  it's a really wide-spread modification.

Reply:
If this is a common pattern we can delay replacing it.